model-inference/decisionTree/experiments/convert_trained_model_to_framework.py

```diff
@@ -143,7 +156,24 @@ def convert_to_treelite_model(model, config):
         # print("predictions", pred_onx[0])
         # # print("predict_proba", pred_onx[1][:1])
     else:
-        print(f"TreeLite models only supports xgboost algorithm, but does not support {MODEL} algorithm.")
+        print(f"TreeLite only supports conversion from xgboost, lightgbm and sk-learn based models. Does not support {MODEL}.")
+
+
+def convert_to_lleaves_model(model, config):
```

Comment:
Where is this function called?

Reply:
Added the reference code for this function.